onlineteamscan_v2_fe/pages/teamdetail/_teamdetail.vue

```diff
@@ -145,29 +149,35 @@ export default {
       ],
       nameRules: [
         value => !!value || 'Vereist',
-        v => /^[a-zA-Z]+$/.test(v) || 'Moet geldig zijn',
+        v => /^[^-\s][ áàíóúéëöüñÄĞİŞȘØøğışÐÝÞðýþa-zA-Z_\s-]*$/.test(v) || 'Moet geldig zijn',
```

Comment:
Regex gebruiken is top. Maar het is onleesbaar voor zij die regex niet kennen.
Wat je best kan doen is de regex query abstraheren naar een const die benoemt wat het doet.

const regexForValidLetters = [a-zA-Z] is een simpel voorbeeld.
zo wordt je code leesbaarder voor de volgende